backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java

```diff
@@ -2251,54 +2261,148 @@ public void deleteLogVersionedInputs(Session session, Long repoId, String commit
     }
   }
 
+  private Optional<ArtifactEntity> getExperimentRunArtifact(
+      Session session, String experimentRunId, String key) {
+    ExperimentRunEntity experimentRunObj = session.get(ExperimentRunEntity.class, experimentRunId);
+    if (experimentRunObj == null) {
+      LOGGER.warn(ModelDBMessages.EXP_RUN_NOT_FOUND_ERROR_MSG);
```

Comment:
Is the `experimentRunId` and `key` coming from the user? If so, then a bad user input is not a warning for us.

Reply:
fixed